diff --git a/dist/rich-text/detection.js b/dist/rich-text/detection.js
index 94df1dd22bd25c9e07908148b04d3c02b7e1e2b7..ca658e5de0416cf2c3c75a2589943564f45ce2d7 100644
--- a/dist/rich-text/detection.js
+++ b/dist/rich-text/detection.js
@@ -1,4 +1,4 @@
-import TLDs from 'tlds' with { type: 'json' };
+import TLDs from 'tlds';
 import { graphemeLen } from '@atproto/common-web';
 import { CASHTAG_REGEX, MENTION_REGEX, TAG_REGEX, TRAILING_PUNCTUATION_REGEX, URL_REGEX, } from './util.js';
 export function detectFacets(text) {
