diff --git a/dist/rich-text/detection.js b/dist/rich-text/detection.js
index 01302ad7c2fd8d49518fa1408c4d4d56ee8bc651..3e37c3b05d33c261f2be51c4d1f6140e50fb9a4d 100644
--- a/dist/rich-text/detection.js
+++ b/dist/rich-text/detection.js
@@ -1,4 +1,4 @@
-import TLDs from 'tlds' with { type: 'json' };
+import TLDs from 'tlds';
 import { CASHTAG_REGEX, MENTION_REGEX, TAG_REGEX, TRAILING_PUNCTUATION_REGEX, URL_REGEX, } from './util.js';
 export function detectFacets(text) {
     let match;
