diff --git a/src/use-toast-animations.ts b/src/use-toast-animations.ts
index 6da145f28a8c505e7e4326f0b6d604845c84015b..308946486610060d3d134b697bc8012fa549dcdc 100644
--- a/src/use-toast-animations.ts
+++ b/src/use-toast-animations.ts
@@ -73,7 +73,6 @@ export const useToastAnimations = ({
   const absolutePositionStyle = useAnimatedStyle(() => {
     const base: Record<string, unknown> = {
       position: 'absolute',
-      width: '100%',
       transform: [
         { translateY: yPosition.value },
         { scaleX: stackScaleX.value },
